src/czbenchmarks/tasks/single_cell/perturbation_expression_prediction.py

```diff
@@ -0,0 +1,322 @@
+from typing import List, Dict, Literal
+import pandas as pd
+import numpy as np
+import logging
+from ..task import Task, TaskInput, TaskOutput
+from ...tasks.types import CellRepresentation
+from ...metrics import metrics_registry
+from ...metrics.types import MetricResult, MetricType
+from ..utils import binarize_values
+from ...constants import RANDOM_SEED
+from scipy import sparse as sp_sparse
+
+logger = logging.getLogger(__name__)
+
+
+class PerturbationExpressionPredictionTaskInput(TaskInput):
+    """Pydantic model for PerturbationTask inputs."""
+
+    de_results: pd.DataFrame
+    masked_adata_obs: pd.DataFrame
+    var_index: pd.Index
+    target_genes_to_save: Dict[str, List[str]]
+
+
+class PerturbationExpressionPredictionOutput(TaskOutput):
```

Comment:
Just a nit and out-of-scope for this PR, but just realized all of the tasks' input types are named `*TaskInput` and all output types are named just `*Output`. Oh well. Worth a separate issue to fix this? 😄

Reply:
I made an issue here: https://github.com/chanzuckerberg/cz-benchmarks/issues/376